client/src/components/ImageSelectModal.js

```diff
@@ -43,6 +52,18 @@ export default class ImageSelectModal extends Component {
                 icon: "warning",
             });
         }
+        if (this.checkFileSizeLimit(file)) {
+            this.setState({
+                imageSize: true
```

Comment:
We could not introduce a new variable here, and simply set imageFile to null here